src/geometry/ArborX_HyperSphere.hpp

```diff
@@ -56,7 +56,8 @@ struct ArborX::GeometryTraits::dimension<
   static constexpr int value = DIM;
 };
 template <int DIM>
-struct ArborX::GeometryTraits::tag<ArborX::ExperimentalHyperGeometry::Sphere<DIM>>
+struct ArborX::GeometryTraits::tag<
+    ArborX::ExperimentalHyperGeometry::Sphere<DIM>>
```

Comment:
What is going on here?  clang-format glitch?

Reply:
That's what clang-format is doing for me.